pyproject.toml

```diff
@@ -73,18 +73,24 @@ exclude = [
     'cli/upgrade_helpers/upgrade_helper_v13\.py',                        # 17 - This is legacy code and will not be typed.
     'dataset/sparkdf_dataset\.py',                                       #  3 - This is legacy code and will not be typed.
     'dataset/sqlalchemy_dataset\.py',                                    # 16 - This is legacy code and will not be typed.
+    'core/usage_statistics/anonymizers/batch_anonymizer\.py',            # 10 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/batch_request_anonymizer\.py',    # 16 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/checkpoint_anonymizer\.py',       # 16 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/data_docs_anonymizer\.py',        # 5 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/datasource_anonymizer\.py',       # 9 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/expectation_anonymizer\.py',      # 6 - This code will be removed in 1.0
+    'core/usage_statistics/anonymizers/validation_operator_anonymizer\.py',  # 5 - This code will be removed in 1.0
+    'render/renderer/v3/suite_edit_notebook_renderer\.py',               # 11 - This is legacy code and will not be typed.
+    'render/renderer/v3/suite_profile_notebook_renderer\.py',            # 4 - This is legacy code and will not be typed.
+    'render/renderer/suite_edit_notebook_renderer\.py',                  # 7 - This is legacy code and will not be typed.
+    'render/renderer/suite_scaffold_notebook_renderer\.py',              # 7 - This is legacy code and will not be typed.
+    'render/renderer/datasource_new_notebook_renderer\.py',              # 4 - This is legacy code and will not be typed.
```

Comment:
@NathanFarmer do these look right? Am I missing anything?

Reply:
`great_expectations/render/renderer/notebook_renderer.py` is now typed but should it be deleted when we remove the cli?

Reply:
Just missing: `render/renderer/checkpoint_new_notebook_renderer.py`

Reply:
Good catch. Will update both.

Reply:
I think `notebook_renderer.py` will also be removed with the CLI removal. It is the base class for the other notebook renderers.